lighthouse-core/report/html/report-styles.css

```diff
@@ -128,7 +128,7 @@
   --scorescale-width: 18px;
   --section-padding-vertical: 12px;
   --snippet-background-color: var(--color-gray-50);
-  --snippet-color: var(--color-gray-800);
+  --snippet-color: #0938C2;
```

Comment:
should this be pulled out into a new color variable or does it not matter?

Reply:
Not sure. The color isn't one of the Material Design colors so I didn't think it needed it's own variable.

Reply:
I would like a color variable